src/Features/CSharp/Portable/Completion/CompletionProviders/CrefCompletionProvider.cs

```diff
@@ -422,5 +426,21 @@ private CompletionItemRules GetRules(string displayText)
 
             return Task.FromResult<TextChange?>(new TextChange(selectedItem.Span, insertionText));
         }
+
+        internal TestAccessor GetTestAccessor()
+            => new TestAccessor(this);
+
+        internal readonly struct TestAccessor
+        {
+            private readonly CrefCompletionProvider _crefCompletionProvider;
+
+            public TestAccessor(CrefCompletionProvider crefCompletionProvider)
+            {
+                _crefCompletionProvider = crefCompletionProvider;
+            }
```

Comment:
nit:

```suggestion
             => _crefCompletionProvider = crefCompletionProvider;
```

i actually like that a lot for simple wrapper structs to make it clear this is just for exposing/hiding around a single value.